test/unit/spell.test.js

```diff
@@ -499,13 +499,6 @@ describe('=> Spell', function() {
     );
   });
 
-  it('orHaving with array attr', function() {
-    assert.equal(
-      Post.count().group('authorId').having([ 'count > ?', 10 ]).orHaving('count = 5').toString(),
```

Comment:
`having([])` is not supported in Spell, which is only supported in sequelize adapter with `having: []`